File: src/backend/userServer/models/psicologo.go
package models

type Psicologo struct {
	Utente   Utente   `json:"utente"`
	Pazienti []string `json:"pazienti"`
}

type PsicoInfo struct {
	Nome, Cognome, Email, Citta, Cellulare, Genere string
}

/* func (p *Psicologo) GetInfoPsicologo() *PsicoInfo {
	info := new(PsicoInfo)
	info.Nome = p.Utente.Nome
	info.Cognome = p.Utente.Cognome
	info.Email = p.Utente.Email
	info.Cellulare = p.Utente.Cellulare
	info.Genere = p.Utente.Genere

	return info
} */

File: src/frontend/frontend-apl/src/app/interfaces/psicologo.ts
export interface Psicologo {
    utente: {
        CodFisc: string;
        Cellulare: string;
        Citta: string;
        Nome: string;
        Cognome: string;
        Email: string;
        Genere: string;
        Password?: string;
    };
    pazienti?: string[];
}